python/tests/kat/t_active_health_check.py

```diff
@@ -0,0 +1,140 @@
+import time
+from typing import Generator, Tuple, Union
+
+from abstract_tests import AmbassadorTest, HealthCheckServer, Node
+from kat.harness import Query
+
+"2022-10-04T23:24:45Z"
+"2022-10-04T23:26:45Z"
+
+
+class ActiveHealthCheckTest(AmbassadorTest):
+    def init(self):
+        self.target = HealthCheckServer()
+
+    def config(self) -> Generator[Union[str, Tuple[Node, str]], None, None]:
+        yield self, self.format(
+            """
+---
+apiVersion: getambassador.io/v3alpha1
+kind: Mapping
+name:  {self.target.path.k8s}-health
+hostname: "*"
+prefix: /healthcheck/
+service: {self.target.path.fqdn}
+resolver: endpoint
+load_balancer:
+  policy: round_robin
+health_checks:
+- unhealthy_threshold: 1
+  interval: 1s
+  health_check:
+    http:
+      path: /healthcheck/actualcheck/
```

Comment:
since we are only picking a path, I'm assuming the host is the `service` correct?

Reply:
Yep, it will make a request to the same service and that is currently not configurable.